docs/breaking-changes.md

```diff
@@ -14,6 +14,11 @@ This document uses the following convention to categorize breaking changes:
 
 ## Planned Breaking API Changes (36.0)
 
+### Utility Process unhandled rejection behavior change
+
+Utility Processes will now warn with an error message when an unhandled
+rejection occurs instead of crashing the process.
```

Comment:
Should we add a code snippet to explain how to add an unhandled rejection handler and crash the utility process, if people want to restore the old behavior?

Reply:
@itsananderson done!